packages/lib/src/components/internal/RedirectButton/RedirectButton.tsx

```diff
@@ -16,6 +16,10 @@ function RedirectButton({ label = null, icon = null, payButton, onSubmit, amount
         return `${i18n.get('continueTo')} ${name}`;
     };
 
+    if (!props.showPayButton) {
```

Comment:
I don't see the redirect button in some payments anymore in the playground dropin, such as stored cashapppay, stored ach, and wechat pay (just by checking it out and running playground). 
I think it's because not all pms are using the `RedirectButton` the same way, some PMs pass the `showPayButton` prop to the `RedirectButton` and some are not.

Reply:
Indeed that was the case. Thank you for checking.
I have added some types, it shouldn't be possible to happen again.